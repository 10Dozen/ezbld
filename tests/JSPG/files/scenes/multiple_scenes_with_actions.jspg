# SceneOne

Description for SceneOne.

@ Action One

Action one for SceneOne.

@ Action Two

Action two for SceneOne.

@ Action Three

# AnotherScene

@ Action One
Action one for AnotherScene.

@ Action Two
Action two for AnotherScene.

@ Action Three