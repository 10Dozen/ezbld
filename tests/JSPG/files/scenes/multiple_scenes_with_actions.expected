[0]
cls = JSPGScene
name = SceneOne
desc = Description for SceneOne.

[1]
cls = JSPGAction
name = Action One
scene = SceneOne
desc = Action one for SceneOne.

[2]
cls = JSPGAction
name = Action Two
scene = SceneOne
desc = Action two for SceneOne.

[3]
cls = JSPGAction
name = Action Three
scene = SceneOne

[4]
cls = JSPGScene
name = AnotherScene

[5]
cls = JSPGAction
name = Action One
scene = AnotherScene
desc = Action one for AnotherScene.

[6]
cls = JSPGAction
name = Action Two
scene = AnotherScene
desc = Action two for AnotherScene.

[7]
cls = JSPGAction
name = Action Three
scene = AnotherScene
